packages/trader/src/AppV2/Containers/ContractDetails/contract-details.tsx

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import EntryExitDetails from 'AppV2/Components/EntryExitDetails';
+import TakeProfitHistory from 'AppV2/Components/TakeProfitHistory';
+import PayoutInfo from 'AppV2/Components/PayoutInfo';
+import ChartPlaceholder from '../Chart';
+import CardWrapper from 'AppV2/Components/CardWrapper';
+import { observer, useStore } from '@deriv/stores';
+import useContractDetails from 'AppV2/Hooks/useContractDetails';
+import OrderDetails from 'AppV2/Components/OrderDetails';
+import { getContractDetailsConfig } from 'AppV2/Utils/contract-details-config';
+import TakeProfit from 'AppV2/Components/TakeProfit/take-profit';
+import StopLoss from 'AppV2/Components/StopLoss/stop-loss';
+import DealCancellation from 'AppV2/Components/DealCancellation/deal-cancellation';
+import {
+    isForwardStarting,
+    isMultiplierContract,
+    isOpen,
+    isValidToCancel,
+    WS,
+    TContractStore,
+    isValidToSell,
+    hasContractEntered,
+    isAccumulatorContract,
+} from '@deriv/shared';
+import classNames from 'classnames';
+import ContractDetailsFooter from 'AppV2/Components/ContractDetailsFooter';
+import { ContractCard } from 'AppV2/Components/ContractCard';
+
+const ContractDetails = observer(() => {
+    const { contract_info, is_loading } = useContractDetails();
+    const { contract_id, currency, contract_type, limit_order } = contract_info;
+    const [update_history, setUpdateHistory] = React.useState<TContractUpdateHistory>([]);
+    const { common } = useStore();
+    const { server_time } = common;
+    const { is_take_profit_visible, is_stop_loss_visible } = getContractDetailsConfig(contract_type ?? '');
+    const is_valid_to_sell = isValidToSell(contract_info);
+    type TContractUpdateHistory = TContractStore['contract_update_history'];
+    type TResponse = {
+        contract_update_history: TContractUpdateHistory;
+    };
+
+    const getSortedUpdateHistory = (history: TContractUpdateHistory) =>
+        history.sort((a, b) => Number(b?.order_date) - Number(a?.order_date));
+    const requestUpdatedHistory = React.useCallback((id?: number) => {
+        if (!id) return;
+        WS.contractUpdateHistory(id)
+            .then((response: TResponse) => {
+                setUpdateHistory(getSortedUpdateHistory(response.contract_update_history));
+            })
+            .catch(() => null);
+    }, []);
+
+    React.useEffect(() => {
+        requestUpdatedHistory(contract_id);
+    }, [contract_id, limit_order, requestUpdatedHistory]);
+
+    if (is_loading) return <></>;
+
+    const is_multiplier = isMultiplierContract(contract_info.contract_type);
```

Comment:
May we use `camelCase` in all possible/suitable places?

Reply:
added thanks